drivers/video/vnc/vnc_keymap.c

```diff
@@ -29,13 +29,8 @@
 #include <errno.h>
 
 #include <nuttx/ascii.h>
-
-#define XK_MISCELLANY 1 /* Select X11 character set */
-#define XK_LATIN1     1
-#define XK_XKB_KEYS   1
-
 #include <nuttx/video/vnc.h>
-#include <nuttx/input/x11_keysymdef.h>
+#include <nuttx/input/x11_keysym.h>
```

Comment:
is it intentional to enable more groups for vnc? if yes, can you mention it in the commit message?

Reply:
Here is our plan:

1. Update x11 keysym to the latest version
2. Switch the internal encoding in nuttx/input/kbd_codec.h to x11
3. Remove vnc_keymap.c from the code base
4. Implement sim keyboard on top of PC x11
5. Modify the related component to the new encoding

Since x11 keysym is widely used on many POSIX environment, it make the integration with the high level graphic stack more smooth.

> is it intentional to enable more groups for vnc? if yes, can you mention it in the commit message?

Done.

Reply:
@yamt do you think this PR ready for merge?